src/test/java/com/rakovets/course/java/core/practice/oop/principles/SiameseTest.java

```diff
@@ -0,0 +1,66 @@
+package com.rakovets.course.java.core.practice.oop.principles;
+
+import com.rakovets.course.java.core.practice.oop.principles.cats.Persian;
+import com.rakovets.course.java.core.practice.oop.principles.cats.Person;
+import com.rakovets.course.java.core.practice.oop.principles.cats.Siamese;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SiameseTest {
+
+    @Test
+    void mew() {
+        assertEquals  ("meeuw", new Siamese("Kuzya").mew());
+    }
+
+    static Stream<Arguments> mewProviderArguments() {
+        return Stream.of(
+                Arguments.of(35, new Person(70))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("mewProviderArguments")
+    void mew(double expectedDouble, Person person) {
+        Siamese siamese = new Siamese("Kuzya");
+        siamese.mew(person);
+        assertEquals(expectedDouble, person.getHappiness());
```

Comment:
"Violation: There isn't separating empty lines for GIVEN, WHEN, THEN parts
Solution: Add blank line"